### DISCUSSION TITLE
What is the format of registry-config for authentication?

### DISCUSSION THREAD

**Discussion by cchengleo** [oras-project/oras#707] [Q&A]
When specifying authentication through --registry-config, what is the correct format for that? Could you share an example?

**Comment by qweeah** [comment]
1) ORAS is designed to be config-file-free, the registry config follows schema of [docker credential store files](https://github.com/docker/cli/blob/master/cli/config/configfile/file.go#L18)
2) Actually, oras users don't need to build up a config file on their own. `oras login` command will take care of the specified file correctly. If `--registry-config` is not provided, it will work on `$HOME/.docker/config.json`.
